### ISSUE TITLE
zip file dowloaded from kiosk unreadable

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/kiosk#1309]
Laurel's 13-inch here, and my new export won't work:
![image](https://user-images.githubusercontent.com/49291909/150414484-1185476b-c0c0-4b4d-a752-6c87bd98ddfc.png)

**Comment by urapadmin** [comment]
still works properly on Windows. So that is a particular thing with the Archive Utility. I have to do some research.

**Comment by luizaogs** [comment]
Weird, this worked for me. OS Monterey (12.0.1), Pro 14-inch 2021.

**Comment by lbestock** [comment]
Also weird, another ZIP file I just downloaded (unrelated to kiosk) opened just fine on the same computer.

**Comment by urapadmin** [comment]
what MacOs version are you on, Laurel?

**Comment by lbestock** [comment]
Ah ha, I am on Big Sur (a lovely place in California that is on fire right now) 11.6. Time for an upgrade.

**Comment by urapadmin** [comment]
Ja, try again after the update and if it is solved then, we close this.

**Comment by lbestock** [comment]
Works just fine now.

**Comment by urapadmin** [comment]
I reopen this issue because I have the same issue now on Windows. The built-in Zip handling of Windows cannot extract anything from that ZIP file. 7Zip can, though. I might be using a somewhat too sophisticated compression method...

**Comment by urapadmin** [comment]
ha. I anticipated that already when I developed this. The default compression method is "LZMA". But it can be overridden by a config key:
```yaml
fileexportworkstation
  FileExportTargetZip
    compression_method: "BZIP2"
```
possible methods are: "BZIP2" and "DEFLATED".
I should try if Windows works better with BZIP2 and see what that does to Zip size.

**Comment by urapadmin** [comment]
this was a combination of an error that prevented kiosk from reading the configuration of the fileexportworkstation AND Windows, which supports only one compression method: "DEFLATED", even still on Windows 10 after all those decades. That's poor Microsoft! 

So if you want your Zip files compatible with Windows 10, the config must use "DEFLATED" as the compression method. I have made this the default for kiosk now.

**Comment by urapadmin** [comment]
### @test  kiosk 0.15.1
- DEFLATED is now the standard compression method for ZIP files exported by the FileExportWorkstation. 
- Should work on Mac and Windows.

**Comment by urapadmin** [comment]
I have updated the section in the developer manual.

**Comment by luizaogs** [comment]
Worked on my Mac.

**Comment by urapadmin** [comment]
sorry, I forgot to actually configure Beset with "DEFLATED". I did it now. So please try again.

**Comment by luizaogs** [comment]
Still works.
